### PR TITLE
[IMPROVE] Prevent "App user" from being deleted by the admin

### PR DESCRIPTION
**Pull request by lolimay**
Prevent deleting an app user from the admin - users panel unexpectedly, which can make sure the app user is removed only through the corresponding app uninstallation.

### CODE REVIEW

File: server/methods/deleteUser.js [hunk 1]
```diff
@@ -28,6 +28,12 @@ Meteor.methods({
 			});
 		}
 
+		if (user.type === 'app') {
+			throw new Meteor.Error('App_Deleting_App_User_Error', 'Deleting app user is not allowed', {
```
Comment (sampaiodiego) [inline server/methods/deleteUser.js]:
```suggestion
			throw new Meteor.Error('error-cannot-delete-app-user', 'Deleting app user is not allowed', {
```

File: packages/rocketchat-i18n/i18n/zh.i18n.json [hunk 2]
```diff
@@ -337,6 +337,7 @@
   "Apiai_Key": "Api.ai Key",
   "Apiai_Language": "Api.ai 语言",
   "App_author_homepage": "作者主页",
+  "App_Deleting_App_User_Error": "不允许直接删除App用户”，请卸载相应的App以移除此用户。",
```
Comment (sampaiodiego) [inline packages/rocketchat-i18n/i18n/zh.i18n.json]:
```suggestion
  "error-cannot-delete-app-user": "不允许直接删除App用户”，请卸载相应的App以移除此用户。",
```

File: packages/rocketchat-i18n/i18n/en.i18n.json [hunk 3]
```diff
@@ -345,6 +345,7 @@
   "Apiai_Key": "Api.ai Key",
   "Apiai_Language": "Api.ai Language",
   "App_author_homepage": "author homepage",
+  "App_Deleting_App_User_Error": "Deleting app user is not allowed, uninstall the corresponding app to remove it.",
```
Comment (sampaiodiego) [inline packages/rocketchat-i18n/i18n/en.i18n.json]:
```suggestion
  "error-cannot-delete-app-user": "Deleting app user is not allowed, uninstall the corresponding app to remove it.",
```

**Comment by lolimay** [comment]
@sampaiodiego Thanks!

**Pull request merged**